src/pages/Detail/Modal/ModalShare.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+const ModalShare = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const [success, setSuccess] = useState(false);
+
+  const textInput = useRef();
+
+  const copy = () => {
+    const copyText = textInput.current;
+    copyText.select();
+    document.execCommand('copy');
+    setSuccess(true);
+  };
+
+  const closeModal = () => {
+    setOpenModal(false);
+    setSuccess(false);
+  };
+
+  return (
+    <AllContainer>
+      <BtnShare
+        onClick={() => {
+          setOpenModal(true);
+        }}
+        alt=""
+        src={process.env.PUBLIC_URL + 'images/icon_share.svg'}
```

Comment:
```suggestion
        src='/images/icon_share.svg'
```
이렇게도 해도 적용될 것 같습니다! 혹시 PUBLIC_URL을 따로 기입하신 이유가 있나요?

Reply:
1차 때 적용이 잘 안되었는데, 저걸 붙이니 잘 되더라고요. 
그런데 지금 코드 수정해보니까 저 부분 없이도 잘 구현 되네요...